index.html

```diff
@@ -7,5 +7,18 @@
   </head>
   <body>
 
+    <header>
+      <h1>Chris McNeese</h1>
```

Comment:
Remember to use `float` on the left and right sides of navbars to get a the split styling.